build-logic-commons/code-quality/src/main/kotlin/gradlebuild/testcleanup/TestFilesCleanupRootPlugin.kt

```diff
@@ -44,7 +40,7 @@ class TestFilesCleanupRootPlugin : Plugin<Project> {
             val testFilesCleanupService = project.gradle.sharedServices.registerIfAbsent("testFilesCleanupBuildService", TestFilesCleanupService::class.java) {
                 parameters.rootBuildDir.set(project.buildDir)
                 parameters.projectStates.putAll(globalExtension.projectStates)
-                parameters.cleanupRunnerStep.set(this@whenReady.allTasks.filterIsInstance<KillLeakingJavaProcesses>().isNotEmpty())
+                parameters.cleanupRunnerStep.set(this@whenReady.allTasks.any { it::class.java.name.contains("KillLeakingJavaProcesses") })
```

Comment:
Can we use the extension as well to propagate this value? So the build-logic build and the main build could do something different here?

Reply:
Done.